lib/sqlalchemy/sql/compiler.py

```diff
@@ -5190,7 +5190,9 @@ def fetch_clause(
                 and select._simple_int_clause(fetch_clause)
             ):
                 fetch_clause = fetch_clause.render_literal_execute()
```

Comment:
please revert all changes to compiler.py.  this file must not be modifed in this PR